docker/scout/scripts/addresses.py

```diff
@@ -575,7 +575,24 @@
         "BSMM_WFTM_SCREAM_ECO": '0x77e803F7d0B6d85731F68CfE1f1CA4c48de105F4',
         "BSMM_WFTM_RENBTC_ECO": '0x609A51e13fd0896c0c02f0B642756533b29f36Ab',
         "BSMM_WFTM_TOMB_ECO": '0x3f96f8cA67AcDE9041AD62125C5a4D062b07872c',
-    }
+        "TOMB": '0x6c021ae822bea943b2e66552bde1d2696a53fbb7',
+        "GEIST": '0xd8321AA83Fb0a4ECd6348D4577431310A6E0814d',
+        "MIM": '0x82f0B8B456c1A451378467398982d4834b6829c1',
+        "USDC": '0x04068DA6C83AFCFA0e13ba15A6696662335D5B75',
+        "RENBTC": '0xDBf31dF14B66535aF65AaC99C32e9eA844e14501',
+        "3poolV2": '0x2dd7C9371965472E5A5fD28fbE165007c61439E1',
```

Comment:
ON eth we call 3pool simply 3crv.  So maybe just call this 3crv or 3crv2 or something.  If V1 is retired/no one uses it/it has no AUM you can just call 3crv 3crv I think.